vtex/faststore
# feat: Migrate BannerNewsletter to app folder

## What's the purpose of this pull request?

Migrate the `BannerNewsletter` section to the app folder. 

## How it works?

![chrome-capture-2024-9-6](https://github.com/user-attachments/assets/927ed981-47d6-49ce-9836-de3eb820f8b1)

## How to test it?

Add the BannerNewsletter section in the home:
https://storeframework.myvtex.com/admin/new-cms/faststore/home/edit/4662a615-63d8-11ef-b37f-92769114c624

Build the app again e see the section.
